platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/PlatformEvent.java

```diff
@@ -227,8 +227,12 @@ public Instant getTimeCreated() {
 
     @Nullable
     @Override
-    public SoftwareVersion getSoftwareVersion() {
-        return hashedData.getSoftwareVersion();
+    public SemanticVersion getSoftwareVersion() {
+        if (hashedData.getSoftwareVersion() != null) {
```

Comment:
i dont think the version should ever be null

Reply:
done